phpunit/block-template-utils-test.php

```diff
@@ -1,251 +0,0 @@
-<?php
-/**
- * Tests_Block_Template_Utils class
- *
- * @package WordPress
- */
-
-class Tests_Block_Template_Utils extends WP_UnitTestCase {
-	public function set_up() {
-		parent::set_up();
-		switch_theme( 'emptytheme' );
-		register_post_type(
-			'custom_book',
-			array(
-				'public'       => true,
-				'show_in_rest' => true,
-			)
-		);
-		register_taxonomy( 'book_type', 'custom_book' );
-	}
-
-	public function tear_down() {
-		unregister_post_type( 'custom_book' );
-		unregister_taxonomy( 'book_type' );
-		parent::tear_down();
-	}
-
-	public function test_get_template_hierarchy() {
-		$hierarchy = get_template_hierarchy( 'front-page' );
```

Comment:
According to past self, `get_template_hierarchy` has been in Core since 6.2 😄 

https://github.com/WordPress/gutenberg/pull/53451/commits/ce5136ab3af41345aa3dc8ff1095bca9c6499785

As for the tests, I can't remember, but I left them there because I saw a few differences.

Looking at them again, I think the differences might be just the way the test is structured? 

https://github.com/WordPress/wordpress-develop/blame/50af5dbed1ddda38a4033c805d877d552e83e7b9/tests/phpunit/tests/block-templates/getTemplateHierarchy.php#L39

cc @ntsekouras / @jorgefilipecosta to confirm

Reply:
The tests are fine in core and are safe to remove here.